dashboard/app/controllers/profanity_controller.rb

```diff
@@ -8,13 +8,15 @@ class ProfanityController < ApplicationController
   # @returns [Array<String>|nil] Profane words within the given string
   def find
     profanity_result = []
-    if params[:should_cache]
-      cache_key = "profanity/#{params[:language] || request.locale}/#{params[:text] || ''}"
-      profanity_result = Rails.cache.fetch(cache_key) do
-        ProfanityFilter.find_potential_profanities(params[:text] || "", params[:language] || request.locale)
+    unless params[:text].nil?
+      if params[:should_cache]
+        cache_key = "profanity/#{params[:language] || request.locale}/#{params[:text] || ''}"
+        profanity_result = Rails.cache.fetch(cache_key) do
```

Comment:
Is it okay to have any form of user-submitted text as our cache key?  Any issues with emoji etc.?

Reply:
it should be fine, I've tested it with a bunch of alphabets and it just reads the emoji description when you provide that.